happi/cli.py

```diff
@@ -208,8 +219,8 @@ def happi_cli(args):
                     info.enforce_value(item_value)
                     valid_value = True
                     kwargs[info.key] = item_value
-                except Exception:
-                    logger.info(f'Invalid value {item_value}')
+                except Exception as e:
+                    logger.info('Invalid value %s, %s', item_value, e)
```

Comment:
another way to log exceptions that you may want to try is:
```
logger.info(text, exc_info=True)
```
This logs the full traceback, which sometimes is what you want.

Reply:
This is great, thank you!